package.json

```diff
@@ -27,17 +27,11 @@
     "build": "lerna run build",
     "bundle": "lerna run bundle",
     "watch": "lerna run --parallel watch",
-    "start": "(cd electron-app && yarn start)",
-    "start:debug": "(cd electron-app && yarn start --log-level=debug)",
     "test": "lerna run test",
-    "package": "(cd electron-app && yarn package)",
-    "deploy": "(cd electron-app && yarn deploy)",
-    "package:preview": "(cd electron-app && yarn package:preview)",
-    "download:plugins": "(cd electron-app && yarn download:plugins)"
+    "electron": "yarn --cwd applications/electron"
```

Comment:
It may not be obvious to use the following to start the application
`yarn electron start` 

I am wondering what possibilities we have, may be just a reference comment to the application/electron/package.json to see available options.

Reply:
It is in the README.md?